app.py

```diff
@@ -8,157 +10,216 @@
 app.config.from_object("config")
 
 TEMPLATES = {
-    'U':"templateU.bin",
```

Comment:
What's with all the formatting changes? That doesn't have anything to do with Python 3, right? It makes it hard to see what actually changed.

Reply:
oh i think black formater ran on the files. I am very sorry I didn't realize it was active.